frontend/src/pages/CreateRecording.tsx

```diff
@@ -58,57 +94,23 @@ function CreateRecordingPage() {
       <EuiFlexGroup justifyContent={"spaceAround"}>
         <EuiFlexItem grow={false}>
           <div>
-            <EuiButton onClick={() => sendAPICall(nameRef, volumeRef, programRef)}>Create Recording</EuiButton>
+            <EuiButton onClick={() => sendAPICall(name, volume, program)}>Create Recording</EuiButton>
```

Comment:
As above, remove function parameters! You can also write this as `onClick={sendAPICall}`